features/features.go

```diff
@@ -495,7 +495,7 @@ var (
 				contactPerson("joe").
 				productScope(ocpSpecific).
 				enhancementPR(legacyFeatureGateWithoutEnhancement).
-				enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
+				enableForClusterProfile(SelfManaged, configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade, configv1.Default).
```

Comment:
@joelanford  I do not see `SelfManaged` in other feature gates. I am not sure why we need it here.

Reply:
This is supposed to make sure that we are only present in SelfManaged clusters. See https://github.com/openshift/api?tab=readme-ov-file#adding-new-techpreview-featuregate-to-all-only-hypershift, which is the closest example for what we're trying to do.

Reply:
I guess the definition is here: https://github.com/openshift/api/blob/master/features/util.go#L34-L38
```console
	Hypershift         = ClusterProfileName("include.release.openshift.io/ibm-cloud-managed")
	SelfManaged        = ClusterProfileName("include.release.openshift.io/self-managed-high-availability")
	AllClusterProfiles = []ClusterProfileName{Hypershift, SelfManaged}
```